src/picterra/__main__.py

```diff
@@ -0,0 +1,189 @@
+    train_parser.add_argument("detector", help="ID of a detector", type=str)
+
+    # create the parsers for the "create" command
+    create_parser = subparsers.add_parser('create', help="Create resources")
+    create_subparsers = create_parser.add_subparsers(dest='create')
+    # create detector
+    create_detector_parser = create_subparsers.add_parser(
+        'detector', help="Creates a detector, optionally adding some existing raster to it")
+    create_detector_parser.add_argument("--name", help="Name of the detector", type=str)
+    create_detector_parser.add_argument(
+        "--detection-type", help="Detection type of the detector",
+        type=str, choices=['count', 'segmentation'], default='count')
+    create_detector_parser.add_argument(
+        "--output-type", help="Output type of the detector",
+        type=str, choices=['polygon', 'bbox'], default='polygon')
+    create_detector_parser.add_argument(
+        "--training-steps", metavar='',
+        help="Number of steps while training the detector: an integer in the range 500..40000",
+        type=int, default=500)
+    create_detector_parser.add_argument(
+        "-r", "--raster", help="ID(s) of the raster (s) to associate with the detector",
+        type=str, required=False, nargs='+', default=[])
+    # create raster
+    create_raster_parser = create_subparsers.add_parser(
+        'raster', help="Creates a raster, optionally adding it to a detector")
+    create_raster_parser.add_argument("path", help="Path where retrieving the raster", type=str)
+    create_raster_parser.add_argument("--name", help="Name of the raster", type=str, required=False)
+    create_raster_parser.add_argument(
+        "--folder", help="Id of the folder/project to which the raster will be uploaded",
+        type=str, required=False)
+    create_raster_parser.add_argument(
+        "-d", "--detector", help="ID(s) of the detector(s) to which we'll associate the raster",
+        type=str, required=False, nargs='+', default=[])
+    # create annotation
+    create_annotation_parser = create_subparsers.add_parser(
+        'annotation', help="Add an annotation to a raster for a given detector")
+    create_annotation_parser.add_argument(
+        "path", help="Path where retrieving the annotation geometries", type=str)
+    create_annotation_parser.add_argument("raster", help="ID of a raster", type=str)
+    create_annotation_parser.add_argument("detector", help="ID of a detector", type=str)
+    create_annotation_parser.add_argument(
+        "type", help="Type of the annotation", type=str,
+        choices=['outline', 'training_area', 'testing_area', 'validation_area'])
+    # create detection area
+    create_detection_area_parser = create_subparsers.add_parser(
+        'detection_area', help="Add a detection area to a raster")
+    create_detection_area_parser.add_argument(
+        "path", help="Path where retrieving the detection area geometries",
```

Comment:
nit: what about "Path to the geojson file containing the detection area geometries"?